src/main/java/org/folio/util/ErrorCode.java

```diff
@@ -38,7 +38,9 @@
   ERROR_FIELDS_MAPPING_INVENTORY("error.mapping.fieldsMappingInventory", "An error occurred during fields mapping for inventory record"),
   ERROR_FIELDS_MAPPING_INVENTORY_WITH_REASON("error.mapping.fieldsMappingInventoryWithReason", "An error occurred during fields mapping for inventory record, reason: %s, cause: %s"),
   ERROR_FIELDS_MAPPING_SRS("error.mapping.fieldsMappingSrs", "An error occurred during fields mapping for srs record with id: %s, reason: %s, cause: %s"),
-  ERROR_MESSAGE_PLACEHOLDER("error.messagePlaceholder", "%s");
+  ERROR_MESSAGE_PLACEHOLDER("error.messagePlaceholder", "%s"),
+  MAPPING_PROFILE_MISSING("error.mappingProfile.missing", "Mapping profile is missing during export process"),
+  INVALID_SRS_MAPPING_PROFILE_RECORD_TYPE("error.mappingProfile.invalidSrsRecordTypeCombination", "SRS record type cannot be combined together with INSTANCE record type during export process");
```

Comment:
Ideally this check should happen when saving the mapping profile itself not during the export, if the change is too much to make in this PR please create a small story

Reply:
Ok, I will create a little user story

Reply:
Created the correspondent user story https://issues.folio.org/browse/MDEXP-356